### PR TITLE
Switch from Thin to Puma

### PR DESCRIPTION
**Pull request by timurvafin**
* Commit database.yml to git to setup pool size
* Set `PORT` var, make sure to update your local `.env`

https://devcenter.heroku.com/articles/deploying-rails-applications-with-the-puma-web-server

Closes: https://github.com/fs/rails-base/issues/286

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - Add gem [rack-canonical-host](https://github.com/tylerhunt/rack-canonical-host) for the definition a single host name as the canonical host for your application.
 - Add Google Analytics
 - Add ability to close app from unauthorized viewers, see: https://github.com/fs/rails-base/pull/302/files
+- Switch web server from [Thin](https://github.com/macournoyer/thin) to [Puma](https://github.com/puma/puma)
```
Comment (vast) [inline CHANGELOG.md]: wrong place for the note

**Comment by vast** [comment]
please fix the message in changelog and :shipit:

**Comment by timurvafin** [comment]
@vast done

**Comment by vast** [comment]
ok

**Pull request merged**